modules/linear_boltzmann_solvers/discrete_ordinates_curvilinear_solver/lbs_curvilinear_solver.cc

```diff
@@ -244,87 +206,71 @@ DiscreteOrdinatesCurvilinearSolver::PerformInputChecks()
           }
         }
         if (not face_orthogonal)
-        {
-          log.LogAllError() << "D_DO_RZ_SteadyState::SteadyStateSolver::PerformInputChecks : "
-                            << "mesh contains boundary faces not orthogonal with respect to "
-                            << "Cartesian reference frame.";
-          Exit(EXIT_FAILURE);
-        }
+          throw std::logic_error("DiscreteOrdinatesCurvilinearSolver::PerformInputChecks : "
+                                 "Mesh contains boundary faces not orthogonal with respect to"
+                                 "Cartesian reference frame.");
       }
     }
   }
-
-  log.Log() << "D_DO_RZ_SteadyState::SteadyStateSolver::PerformInputChecks : exit";
+  log.Log() << "DiscreteOrdinatesCurvilinearSolver::PerformInputChecks : exit";
 }
 
 void
 DiscreteOrdinatesCurvilinearSolver::InitializeSpatialDiscretization()
 {
   log.Log() << "Initializing spatial discretization_.\n";
 
-  auto qorder = QuadratureOrder::INVALID_ORDER;
-  auto system = CoordinateSystemType::UNDEFINED;
-
-  //  primary discretisation
+  //  primary discretization
```

Comment:
You can fix the double space here as well (I saw you fixed it above). In case you care...